Src/CSharpier.VSCode/src/InstallerService.ts

```diff
@@ -11,22 +12,34 @@ export class InstallerService {
     globalErrorVisible = false;
     logger: Logger;
     killRunningProcesses: () => void;
+    extension: Extension<unknown>;
 
-    constructor(logger: Logger, killRunningProcesses: () => void) {
+    constructor(logger: Logger, killRunningProcesses: () => void, extension: Extension<unknown>) {
         this.logger = logger;
         this.killRunningProcesses = killRunningProcesses;
+        this.extension = extension;
     }
 
     public displayInstallNeededMessage = (directoryThatContainsFile: string) => {
         this.logger.error("CSharpier was not found so files may not be formatted.");
 
-        const items = ["Install CSharpier Globally"];
+        this.logger.info(this.extension.extensionKind);
+
+        if (this.extension.extensionKind === vscode.ExtensionKind.Workspace) {
+            window.showErrorMessage("CSharpier must be installed as a dotnet tool on the local host using `dotnet tool install csharpier` and the devcontainer rebuilt to support formatting files.");
```

Comment:
What exactly do you mean by 'local host'? Machine outside the dev container?

Reply:
Yeah. VSCode uses that verbiage. Not sure if just calling it "local machine" or "host" would make it more clear. Maybe local machine because no one really refers to a container as a machine, but a container can host other things.

Reply:
another way to say it I guess, is there must be a dotnet tools manifest in the directory that includes csharpier. Hmmm. Maybe that would be more clear.